test-e2e/utils.js

```diff
@@ -511,6 +513,40 @@ async function seedProjectDatabase(
   return Promise.all(promises)
 }
 
+/**
+ * If the path is a regular file, return its size.
+ *
+ * If the path is a directory, return the size of all regular files inside.
+ * Recurses through subdirectories.
+ *
+ * If the path is anything else, such as a symlink, `0` is returned.
+ *
+ * @param {string} filePath
+ * @returns {Promise<number>}
+ */
+export async function getFileSize(filePath) {
```

Comment:
surprised their isn't an easier way to do this

nit: rename to something slightly more generic e.g. `getDiskUsage`